includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -934,4 +940,53 @@ public static function wpseo_og_url( $og_url ) {
 
 		return $og_url;
 	}
+
+	/**
+	 * Updates a post content via a REST request after the new post is created
+	 * in order to get the rendered content.
+	 *
+	 * @param int $new_post_id The new post ID that was pulled.
+	 * @return void
+	 */
+	public function update_content_via_rest( $new_post_id ) {
+
+		$post = get_post( $new_post_id );
+		if ( ! is_a( $post, '\WP_Post' ) ) {
+			return;
+		}
+
+		$original_blog_id = absint( get_post_meta( $post->ID, 'dt_original_blog_id', true ) );
+		$original_post_id = absint( get_post_meta( $post->ID, 'dt_original_post_id', true ) );
+
+		$rest_url = false;
+		if ( ! empty( $original_blog_id ) && ! empty( $original_post_id ) ) {
+			$rest_url = Utils\get_rest_url( $original_blog_id, $original_post_id );
+		}
+
+		if ( empty( $rest_url ) ) {
+			return;
+		}
+
+		$request = apply_filters( 'dt_update_content_via_request_args', [], $new_post_id, $this );
+
+		$response = wp_remote_get( $rest_url, $request );
```

Comment:
To support VIP, I think we'll need to enhance this a bit. Basically we need to check if we are on WordPress.com VIP and if so, use the `vip_safe_wp_remote_get` function. Can see examples of this here: https://github.com/10up/distributor/blob/develop/includes/classes/ExternalConnections/WordPressExternalConnection.php#L184